src/cli/task.rs

```diff
@@ -182,6 +192,31 @@ pub struct Args {
     pub manifest_path: Option<PathBuf>,
 }
 
+fn print_heading(value: &str) {
+    let bold = console::Style::new().bold();
+    eprintln!("{}\n{:-<2$}", bold.apply_to(value), "", value.len(),);
+}
+
+fn print_tasks_per_env(envs: Vec<Environment>) -> io::Result<()> {
+    let mut writer = tabwriter::TabWriter::new(stdout());
+    for env in envs {
```

Comment:
Should this have been `for env in envs.sorted()` or something along those lines? In `pixi` 0.28.1 (the latest release) I see that the tasks are indeed sorted for `pixi task list -s`, but the order in which the environments are shown varies randomly.